src/preferences/dialog/dlgprefcontrols.cpp

```diff
@@ -774,8 +815,14 @@ void DlgPrefControls::slotNumSamplersChanged(double new_count) {
         m_cueControls.push_back(new ControlProxy(
                 group, "cue_mode"));
         m_keylockModeControls.push_back(new ControlProxy(
-                        group, "keylockMode"));
+                group, "keylockMode"));
         m_keylockModeControls.last()->set(m_keylockMode);
+        // Do we need key un-lock mode for Samplers as well?
+        // Except GUI, they probably don't have proper (physical!) controls
+        // to change pitch.
+        m_keyunlockModeControls.push_back(new ControlProxy(
+                group, "keyunlockMode"));
+        m_keyunlockModeControls.last()->set(m_keyunlockMode);
```

Comment:
AFAIK Samplers don't even have controls for key modification in GUI. Maybe someone has a dedicated Sampler controller with those controls mapped.
Should it override config value and be set to default here (1 = reset key on unlock)?